src/sketchup-stl.rb

```diff
@@ -52,7 +52,7 @@ def self.translate(string)
       'This is an open source project sponsored by the SketchUp team. More ' <<
       'info and updates at https://github.com/SketchUp/sketchup-stl'
     )
-    extension.version = '2.1.6'
```

Comment:
Ah - I see. I don't think we published that. So on EW it's still 2.1.6. Can you make a new commit with version set to 2.1.7? Then I'm squash-merge this and get it uploaded.